Makefile

```diff
@@ -46,11 +46,11 @@ coverage:
 		coverage html
 
 container-deploy:
-	docker build -t modulus:deploy --target deploy -f Dockerfile .
+	docker build -t modulus:deploy --build-arg TARGETPLATFORM=linux/amd64 --target deploy -f Dockerfile .
```

Comment:
Would be good to set `TARGETPLATFORM` based on the detected CPU arch on the system. Something like

```make
ARCH := $(shell uname -p)

ifeq ($(ARCH), x86_64)
    TARGETPLATFORM := "linux/amd64"
else
    ifeq ($(ARCH), aarch64)
        TARGETPLATFORM := "linux/arm64"
    else
        $(error Unknown CPU architecture ${ARCH} detected)
    endif
endif

container-deploy:
	docker build -t modulus:deploy --build-arg TARGETPLATFORM=${TARGETPLATFORM} --target deploy -f Dockerfile .
```

Reply:
This is great snippet! Will add this

Reply:
Added